docs/en/observability/cloud-monitoring/azure/monitor-azure-intro.asciidoc

```diff
@@ -0,0 +1,28 @@
+
+[[monitor-azure-web-services]]
+=   Azure monitoring
+
+++++
+<titleabbrev>Azure monitoring</titleabbrev>
+++++
+
+Elastic Observability offers powerful monitoring solutions to keep your Azure environments reliable and efficient, providing deep insights into the performance of your applications, services, and infrastructure components.
+
+Learn how to use the Elastic Observability solution to observe and monitor a broad range of Azure resources and applications. 
+
+- <<monitor-azure-elastic-agent>>
+- <<monitor-azure>>
+- <<monitor-azure-native>>
+
+
+For a full list of supported Azure integrations, check the {integrations-docs}[Elastic
+Integrations docs].
+
+include::monitor-azure-agent.asciidoc[]
+
+include::collect-azure-metrics.asciidoc[leveloffset=+2]
```

Comment:
Do you want to remove this page from here since it's not ready for publication? Otherwise, it shows up in the navigation:

![image](https://github.com/elastic/observability-docs/assets/114418652/013977b1-910a-41f9-b71a-2190d21a5113)

Reply:
Good catch! I'll remove it right away.